pydantic/_internal/_config.py

```diff
@@ -75,6 +75,7 @@ class ConfigWrapper:
     protected_namespaces: tuple[str, ...]
     hide_input_in_errors: bool
     defer_build: bool
+    defer_build_mode: Literal['only_model', 'always']
```

Comment:
Could we call this `experimental_defer_build_mode` or `beta_defer_build_mode`? That way users are aware that there might be breaking changes in minor releases for this feature 👍

Reply:
I see there is another experimental feature in TypeAdapter, the `_parent_depth`. Should this be also underscore prefixed as `_defer_build_mode`?

Reply:
I now pushed `_defer_build_mode` similar to `_parent_depth` experimental TypeAdapter parameter